assets/js/util.js

```diff
@@ -242,8 +243,19 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function callApy(options, endpoint) {
+    return $.ajax(Object.assign({
+        url: config.APY_URL + endpoint,
+        beforeSend: ajaxSend,
+        contentType: 'application/x-www-form-urlencoded; charset=UTF-8',
+        dataType: 'json',
+        type: (options.data.q.length > THRESHOLD_REQUEST_LENGTH) ? 'POST' : 'GET'
```

Comment:
For instance, I ran this example with `$.param()` and `originalText` content as `hello world`. The URL that gets generated is as follows: (I modified `window.location.href` as it gets modified according to the contents of `persistChoices` function for implementation purpose)

This is the output I get: `http://localhost:8080/index.eng.html?langpair=eng%7Cspa&q=hello+world&markUnknown=yes`. I could check the length of this URL and make the decision right?